cms/djangoapps/contentstore/views/import_export.py

```diff
@@ -52,7 +55,7 @@
 @ensure_csrf_cookie
 @require_http_methods(("GET", "POST", "PUT"))
 @ensure_valid_course_key
-def import_handler(request, course_key_string):
+def course_import_handler(request, course_key_string, library=False):
```

Comment:
Same thing as an earlier comment... If this method can work with both courses and libraries, I think you should rename it back to `import_handler` with the signature `def import_handler(request, courselike_key_string)`. Inside this method, after you've gotten `course_key`, you can add `library = isinstance(course_key, LibraryLocator)` - i.e. since you can detect whether it's a course or library from they key argument, there's no reason to require it as a parameter.

For the corresponding entry in urls.py, just change it to `COURSELIKE_KEY_PATTERN` and it will work with both types of key in the url.